test_haystack/core/models.py

```diff
@@ -17,7 +17,7 @@ class MockModel(models.Model):
     author = models.CharField(max_length=255)
     foo = models.CharField(max_length=255, blank=True)
     pub_date = models.DateTimeField(default=datetime.datetime.now)
-    tag = models.ForeignKey(MockTag)
+    tag = models.ForeignKey(MockTag, on_delete=models.CASCADE)
```

Comment:
You can omit `on_delete=` and pass it as a positional argument... that's what we've done in Django's test models anyway.